src/editor/grid.rs

```diff
@@ -184,6 +191,28 @@ mod tests {
         }
     }
 
+    fn assert_grid_cell_equal_to_char(grid: &CharacterGrid, x: usize, y: usize, chr: &str) {
```

Comment:
nit: I try to avoid unnecessary abbreviations because I think it makes the code a little unapproachable. Maybe replace chr with char? Or even better just text since its not really a char. If you do, make sure to rename the function as well to match

Reply:
tbh I might even rename the function to be `assert_grid_cell_contents` which is shorter too!